pkg/models/testrun.go

```diff
@@ -17,8 +17,10 @@ type TestReport struct {
 }
 
 type TestCoverage struct {
-	FileCov  map[string]string `json:"fileCoverage" yaml:"file_coverage"`
-	TotalCov string            `json:"totalCoverage" yaml:"total_coverage"`
+	FileCov   map[string]string `json:"fileCoverage" yaml:"file_coverage"`
+	BranchCov map[string]string `json:"branchCoverage" yaml:"branch_coverage"`
```

Comment:
Does this PR has all the branch coverage related changes ??

Reply:
this got committed by mistake. reverting it

Reply:
this got committed by mistake. reverting it